content/en/guides/models/track/log/image-normalization.md

```diff
@@ -0,0 +1,139 @@
+   ```
+
+3. **For any other range**: Values are clipped to [0, 255] and converted to uint8
+   ```python
+   normalized_data = data.clip(0, 255).astype(np.uint8)
+   ```
+
+## Examples of normalization effects
+
+### Example 1: [0, 1] range data
+
+```python
+import torch
+import wandb
+
+# Create tensor with values in [0, 1] range
+tensor_0_1 = torch.rand(3, 64, 64)  # Random values between 0 and 1
+
+# This will multiply all values by 255
+image = wandb.Image(tensor_0_1, caption="Normalized from [0,1] range")
+```
+
+### Example 2: [-1, 1] range data
+
+```python
+import torch
+import wandb
+
+# Create tensor with values in [-1, 1] range
+tensor_neg1_1 = torch.rand(3, 64, 64) * 2 - 1  # Random values between -1 and 1
+
+# This will rescale: -1 → 0, 0 → 127.5, 1 → 255
+image = wandb.Image(tensor_neg1_1, caption="Normalized from [-1,1] range")
+```
+
+**Note on visual contrast**: The [-1, 1] normalization creates higher visual contrast compared to [0, 1] normalization. This is because:
+- Negative values (like -0.8) become very dark (around 25)
+- Positive values (like 0.8) become very bright (around 230)
+- Values near 0 become mid-gray (127.5)
+
+This "stretches" the visual range, making differences between pixel values more pronounced. This is particularly useful for highlighting subtle patterns in machine learning data, but if you want less contrast, consider preprocessing your data to a [0, 1] range before logging.
+
+### Example 3: Avoiding normalization with PIL Images
+
+```python
+import torch
+from PIL import Image as PILImage
+import wandb
+
+# Create tensor with values in [0, 1] range
+tensor_0_1 = torch.rand(3, 64, 64)
+
+# Convert to PIL Image to avoid normalization
+pil_image = PILImage.fromarray((tensor_0_1.permute(1, 2, 0).numpy() * 255).astype('uint8'))
+image = wandb.Image(pil_image, caption="No normalization applied")
+```
+
+### Example 4: Using normalize=False
+
+```python
+import torch
+import wandb
+
+# Create tensor with values in [0, 1] range
+tensor_0_1 = torch.rand(3, 64, 64)
+
+# Disable normalization - values will be clipped to [0, 255]
+image = wandb.Image(tensor_0_1, normalize=False, caption="Normalization disabled")
+```
+
+## When to use different approaches
+
+### Use PIL conversion when:
+- You want complete control over pixel values
+- You need custom preprocessing (filters, brightness adjustments, etc.)
+- You want to use PIL's image processing capabilities
+- You're debugging and want to see exact values being logged
+
+### Use normalize=False when:
+- You want to see raw tensor values as they are
+- Your data is already in the correct range (like [0, 255] integers)
+- You're debugging normalization issues
+- Quick testing without additional processing steps
+
+### Use automatic normalization when:
+- You want consistent behavior across different input types
+- Your data is in standard ranges ([0, 1] or [-1, 1])
+- You want the system to handle the conversion automatically
+
+## Best practices
```

Comment:
Combine the Best practices sectino with the Common issues section and call it Troubleshooting. You will need to adjust the contents a bit.